api/Navigator.json

```diff
@@ -281,21 +281,35 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/serviceWorker",
           "support": {
+            "webview_android": {
+              "version_added": "40"
+            },
```

Comment:
`webview_android` should be at the end of the `support` block (issues #1662 and #398)

/cc @jpmedley

Reply:
Thanks for the review, done!